src/sentry/integrations/slack/message_builder/event.py

```diff
@@ -1,70 +1,25 @@
-from typing import Mapping
-
 from sentry.features.helpers import any_organization_has_feature
 from sentry.integrations.slack.message_builder import SlackBody
-from sentry.integrations.slack.message_builder.base.block import BlockSlackMessageBuilder
+from sentry.integrations.slack.message_builder.help import SlackHelpMessageBuilder
 from sentry.models import Integration
 
 EVENT_MESSAGE = (
     "Want to learn more about configuring alerts in Sentry? Check out our documentation."
 )
-HEADER_MESSAGE = "Here are the commands you can use. Commands not working? Re-install the app!"
-DM_COMMAND_HEADER = "*Direct Message Commands:*"
-CHANNEL_COMMANDS_HEADER = "*Channel Commands:*"
-GENERAL_MESSAGE = "Just want to learn more about Sentry? Check out our documentation."
-
-DM_COMMANDS = {
-    "link": "Link your Slack account to Sentry",
-    "unlink": "Unlink your Slack account from Sentry",
-    "help": "View this list of commands",
-}
-CHANNEL_COMMANDS = {
-    "link team": "Type this into the channel in which you want your team to receive issue alert notifications"
-}
-
-
-def list_commands(commands: Mapping[str, str]) -> str:
-    return "\n".join(
-        (
-            f"• `/sentry {command}`: {description}"
-            for command, description in sorted(tuple(commands.items()))
-        )
-    )
 
 
-DM_COMMANDS_MESSAGE = list_commands(DM_COMMANDS)
-CHANNEL_COMMANDS_MESSAGE = list_commands(CHANNEL_COMMANDS)
-
-
-class SlackEventMessageBuilder(BlockSlackMessageBuilder):
+class SlackEventMessageBuilder(SlackHelpMessageBuilder):
     def __init__(self, integration: Integration) -> None:
         super().__init__()
         self.integration = integration
 
     def build(self) -> SlackBody:
-        action_block = self.get_action_block(
-            [
-                (
-                    "Sentry Docs",
-                    "https://docs.sentry.io/product/alerts-notifications/alerts/",
-                    "sentry_docs_link_clicked",
-                )
-            ]
-        )
         if not any_organization_has_feature(
```

Comment:
Somewhere in the shuffle this got broken - I don't have the feature turned on and I am getting the newer help text. We still want to feature flag this until it's released because we're telling users to re-install the app if the commands aren't working, but that won't do anything until the feature is released.